wallet/wallet-controller/src/lib.rs

```diff
@@ -813,7 +855,7 @@ impl<T: NodeInterface + Clone + Send + Sync + 'static, W: WalletEvents> Controll
                         tokio::time::sleep(ERROR_DELAY).await;
                     }
 
-                    continue;
+                    continue 'outer;
```

Comment:
Congratulations, I think this is the first usage of this construct in the code base.